README.md

```diff
@@ -20,6 +20,10 @@ Feeling stuck amidst the course? Feel free to refer this repository or reach out
  &#9746; Means that the particular Chapter is under progress 👨‍💻 <br>
  &#9745; Means that the particular Chapter is completed 🎉
  
+ ## How does this scholarship program work?
+ ### Phase 1: Scholarship Foundations Course
```

Comment:
Let's Keep these as bullet points and not the Headings.

Reply:
Changes done.